_rules/disable-shortcut-670a30.md

```diff
@@ -21,21 +21,21 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to any [HTML document][] with at least one [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger an action on a [user interface component][].
+The rule applies to any [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger an [event][] within a [HTML document][].
 
 ## Expectation 1
 
-For each [user interface component][] that is a [descendant][] of the root node of the test target, each [printable character][] [shortcut][keyboard shortcut] that triggers the action of that [user interface component][] can be disabled by another [user interface component][].
+For each test target, an [instrument][] to disable it is available.
 
-**Note:** If multiple single [printable character][] [shortcuts][keyboard shortcut] exist, they all can be disabled by a single [user interface component][].
+**Note:** If multiple single [printable character][] [shortcuts][keyboard shortcut] exist, they all can be disabled by a single [instrument][].
```

Comment:
What do you base this not on? Is there a requirement that all such keyboard short-cuts can be disabled with a single instrument?